openwisp_radius/tests/test_commands.py

```diff
@@ -165,61 +166,64 @@ def test_deactivate_expired_users_command(self):
         self.assertEqual(get_user_model().objects.filter(is_active=True).count(), 0)
 
     @capture_stdout()
-    def test_delete_old_radiusbatch_users_command(self):
-        # Create RadiusBatch users that expired more than 18 months ago
-        path = self._get_path("static/test_batch.csv")
-        options = dict(
-            organization=self.default_org.slug,
-            file=path,
-            expiration="28-01-1970",
-            name="test",
-        )
-        self._call_command("batch_add_users", **options)
-        # Create RadiusBatch users that expired 15 months ago
-        expiration_date = (now() - timedelta(days=30 * 15)).strftime("%d-%m-%Y")
-        path = self._get_path("static/test_batch_new.csv")
-        options = dict(
-            organization=self.default_org.slug,
-            file=path,
-            expiration=expiration_date,
-            name="test1",
-        )
-        self._call_command("batch_add_users", **options)
-        # Create RadiusBatch users that expired 10 days ago
-        path = self._get_path("static/test_batch_users.csv")
-        expiration_date = (now() - timedelta(days=10)).strftime("%d-%m-%Y")
-        options = dict(
-            organization=self.default_org.slug,
-            file=path,
-            expiration=expiration_date,
-            name="test2",
-        )
-        self._call_command("batch_add_users", **options)
-        self.assertEqual(get_user_model().objects.all().count(), 9)
-
-        with self.subTest("Test executing command without arguments"):
-            call_command("delete_old_radiusbatch_users")
-            # Only users that expired more than 18 months ago should be deleted
-            self.assertEqual(get_user_model().objects.all().count(), 6)
-
-        with self.subTest("Test executing command with older_than_months argument"):
-            call_command("delete_old_radiusbatch_users", older_than_months=12)
-            # Users that expired more than 12 months ago should be deleted
-            self.assertEqual(get_user_model().objects.all().count(), 3)
-
-        with self.subTest("Test executing command with older_than_days argument"):
-            call_command("delete_old_radiusbatch_users", older_than_days=9)
-            # Users that expired more than 9 days ago should be deleted
-            self.assertEqual(get_user_model().objects.all().count(), 0)
-
-        with self.subTest("Test executing command with both arguments"):
-            options["name"] = "test3"
-            call_command("batch_add_users", **options)
-            call_command(
-                "delete_old_radiusbatch_users", older_than_days=9, older_than_months=12
+    def test_delete_old_radiusbatch_users_command(self, subtest_id=None):
+        with freeze_time(_TEST_DATE):
```

Comment:
We can use a decorator to the test method which is more consistent with what we have in the rest of the codebase.